### PR TITLE
RD: CommitmentId field for future Elevate Payments integration

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

New Field for Future Use:
* npe03__Recurring_Donation__c.CommitmentId__c

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
**lurch attach W-039004

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrKFQA0" target="_blank">W-039004</a>

File: src/objects/npe03__Recurring_Donation__c.object [hunk 1]
```diff
@@ -63,6 +63,19 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
+        <fullName>CommitmentId__c</fullName>
```
Comment (force2b) [inline src/objects/npe03__Recurring_Donation__c.object]: @sam-knox Would you mind reviewing the new field label, help text, etc. just to make sure it all looks good?

Comment (sam-knox) [reply src/objects/npe03__Recurring_Donation__c.object]: @force2b - We decided to go with unique = true?

**Pull request merged**